arwen/elrondapi/bigIntOps.go

```diff
@@ -345,35 +362,44 @@ func bigIntStorageLoadUnsigned(context unsafe.Pointer, keyOffset int32, keyLengt
 
 	bytes := storage.GetStorage(key)
 
-	value := bigInt.GetOne(destination)
+	value, err := bigInt.GetOne(destinationHandle)
+	if arwen.WithFault(err, context, runtime.BigIntAPIErrorShouldFailExecution()) {
```

Comment:
for this case you need GetOrCreate

Reply:
same for here. destination created before call of function